config-path.js

```diff
@@ -12,7 +12,7 @@ function macos () {
 
 function windows () {
   const appData = env.LOCALAPPDATA || path.join(userHome, 'AppData', 'Local');
-  return path.join(appData, name, 'Cache');
+  return path.join(appData, name);
```

Comment:
Since it's the kind of data this app writes and as Windows isn't opinionated about this, think it's a good solution that prevents the need for a module like `mkdirp`.